lib/dotiw/methods.rb

```diff
@@ -67,19 +92,23 @@ def _display_time_in_words(hash, options = {})
         include_seconds: false
       ).symbolize_keys!
 
+      discarded_hash = {}
+
       include_seconds = options.delete(:include_seconds)
-      hash.delete(:seconds) if !include_seconds && hash[:minutes]
+      discarded_hash[:seconds] = hash.delete(:seconds) if !include_seconds && hash[:minutes]
```

Comment:
We should be `dup`ing `hash` to avoid modifying input.

Reply:
The mutations to input were already here, and are not strictly necessary (since all the callers of this create a new hash just for calling it), but I'm down with defensive programming so I made the change.